cachito/workers/tasks/pip.py

```diff
@@ -118,18 +117,15 @@ def fetch_pip_source(request_id, package_configs=None):
     worker_config = get_worker_config()
     env_vars.update(worker_config.cachito_default_environment_variables.get("pip", {}))
 
-    packages_json_data = PackagesData()
+    update_request_env_vars(request_id, env_vars)
 
+    packages_json_data = PackagesData()
     # Finally, perform DB operations
```

Comment:
Isn't this comment outdated?

Reply:
Good catch. Fixed in a new commit. Thanks.